### PR TITLE
Support "IntOrString"

### PR DESCRIPTION
**Pull request by ityuhui**
This PR consists of two parts:

1. The files generated by [kubernetes-client/gen](https://github.com/kubernetes-client/gen) and [OpenAPITools/openapi-generator](https://github.com/OpenAPITools/openapi-generator)
2. The model implementation for `IntOrString` I created manually.

### CODE REVIEW

File: kubernetes/model/int_or_string.c [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#include <stdio.h>
+#include "int_or_string.h"
+
+int_or_string_t *int_or_string_create()
+{
+    int_or_string_t *ios = calloc(sizeof(int_or_string_t), 1);
+    if (!ios) {
+        return NULL;
+    }
+    ios->type = IOS_DATA_TYPE_INT;
```
Comment (brendandburns) [inline kubernetes/model/int_or_string.c:12]: it's not required, but probably slightly more friendly to set `ios->s = NULL`

**Comment by brendandburns** [comment]
One really minor comment, but LGTM in general. Thanks for handling this!

File: kubernetes/model/int_or_string.c (hunk 1 above)
Comment (ityuhui) [reply kubernetes/model/int_or_string.c:12]: Thank you @brendandburns. It's done.

**Comment by ityuhui** [comment]
Hi @brendandburns 

Could you please take another look ?

**Comment by ityuhui** [comment]
Thank you @brendandburns 

Happy Holidays !

**Pull request merged**